repl/src/main/scala/ammonite/repl/interp/ClassLoaders.scala

```diff
@@ -121,11 +123,42 @@ class SpecialClassLoader(parent: ClassLoader, parentHash: Array[Byte])
     * classloader can get at them.
     */
   val newFileDict = mutable.Map.empty[String, Array[Byte]]
+  val classOutputDir = System.getProperty("ammonite.repl.class.outputdir", null)
```

Comment:
No, `replArgs` is custom value bindings we want to expose to the shell. I just want to add a new constructor parameter next to `replArgs` called whatever you want and plumb it down to where-ever we're using the `SpecialClassLoader`s